src/styles/components/signUpForm.js

```diff
@@ -0,0 +1,67 @@
+import styled from "styled-components";
+import * as styleVars from './variable'
+
+const mailchimpColorMain = styleVars.colorHackRed;
+const mailchimpColorLight = styleVars.colorHackRedMed;
+const mailchimpColorGrey = styleVars.hackBlack70;
+const mailchimpColorWhite = styleVars.hackWhite;
```

Comment:
instead of redefining the colours with new variables, can we just use the `styleVars` variables?